variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "region" {
+  type        = string
+  default     = "fra1"
+  description = "DOKS region (availables regions: `doctl kubernetes options regions`)"
+}
+
+variable "kubernetes_version" {
```

Comment:
Is this used?

Reply:
Yes it is used to get the complete doks version.
We fix kubernetes minor version with this variable, then we use the last digital ocean patch, cf doks_version in locals.tf